misc/release.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
(some day i want to move some of the stuff from `misc` to maybe `bin` or `scripts`.  but not necessarily now i guess.)